src/extension/ext-raw.js

```diff
@@ -15,262 +17,287 @@ const navigationOptions = [
+          type: "string",
+          translation: "Object.OrgChart.FontColor",
+          component: "dropdown",
+          defaultValue: DEFAULTS.FONT_COLOR_TYPE,
+          options: colorOptions,
+        },
+        colorPicker: {
+          component: "color-picker",
+          type: "object",
+          ref: "style.fontColor.color",
+          translation: "properties.color",
+          dualOutput: true,
+          defaultValue: DEFAULTS.FONT_COLOR_DARK,
+          show: (data) => propertyResolver.getValue(data, "style.fontColor.colorType") === "colorPicker",
+        },
+        colorExpression: {
+          component: "string",
+          type: "string",
+          ref: "style.fontColor.colorExpression",
+          translation: "Common.Expression",
+          expression: "optional",
+          defaultValue: "",
+          show: (data) => propertyResolver.getValue(data, "style.fontColor.colorType") === "byExpression",
+        },
+      },
+    };
+
+    settings.items.styling.items.border = {
+      type: "items",
+      items: {
+        appearanceHeader: {
+          component: "text",
+          translation: "Object.OrgChart.CardAppearance",
+          style: "pp-nm-hcd__list-header",
+        },
+        topBar: {
+          type: "boolean",
+          ref: "style.border.top",
+          translation: "Object.OrgChart.TopBar",
+          defaultValue: DEFAULTS.BORDER_TOP,
+        },
+        fullBorder: {
+          type: "boolean",
+          ref: "style.border.fullBorder",
+          translation: "properties.border",
+          defaultValue: DEFAULTS.BORDER_FULL,
+        },
+        colorType: {
+          component: "dropdown",
+          type: "string",
+          ref: "style.border.colorType",
+          translation: "properties.border.color",
+          defaultValue: DEFAULTS.BORDER_COLOR_TYPE,
+          options: colorOptions,
+          show: (data) => bordersActive(data),
+        },
+        colorPicker: {
+          component: "color-picker",
+          type: "object",
+          ref: "style.border.color",
+          translation: "properties.color",
+          dualOutput: true,
+          defaultValue: DEFAULTS.BORDER_COLOR,
+          show: (data) =>
+            bordersActive(data) && propertyResolver.getValue(data, "style.border.colorType") === "colorPicker",
+        },
+        colorExpression: {
+          component: "string",
+          type: "string",
+          ref: "style.border.colorExpression",
+          translation: "Common.Expression",
+          expression: "optional",
+          defaultValue: "",
+          show: (data) =>
+            bordersActive(data) && propertyResolver.getValue(data, "style.border.colorType") === "byExpression",
+        },
+      },
+    };
+  }
+  return settings;
 };
+
+export default function extDef({ translator, flags, anything }) {
+  return {
+    definition: {
+      type: "items",
+      component: "accordion",
+      items: {
+        data: getData(flags),
```

Comment:
The getData function does not seem to have any inputs. Does not have to be a function I think

Reply:
Right, this is the remanent after I split the styling and the images feature into different PRs. The flag will be needed for the image feature. I will remove it now an include it in upcoming PR.